test/distributed/test_functional_api.py

```diff
@@ -1,26 +1,23 @@
 # Owner(s): ["oncall: distributed"]
 
 import sys
+
 import torch
 import torch.distributed as dist
 import torch.distributed._functional_collectives as ft_c
-import torch.distributed.distributed_c10d as c10d
 import torch.distributed._tensor as dt
+import torch.distributed.distributed_c10d as c10d
+from functorch import make_fx
 
 from torch.testing import FileCheck
-from functorch import make_fx
```

Comment:
curious is this ordering what lintrunner gives us?

Reply:
i think lintrunner doesn't always adjust the empty-lines such as line11 above, but it probably insists on `from functorch` coming before `from torch`

Reply:
we should probably change that to `proxy_tensor` import though, surprised `functorch` actually implicitly import make_fx